scripts/sysman/memory.yml

```diff
@@ -159,6 +157,19 @@ members:
             Never take the delta of this timestamp with the timestamp from a different structure since they are not guaranteed to have the same base.
             The absolute value of the timestamp is only valid during within the application and may be different on the next execution.
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension properties for Memory bandwidth"
+version: "1.7"
+class: $sMemory
+name: $s_mem_ext_bandwidth_t
+details:
+    - "Number of counter bits"
+members:
```

Comment:
do we need a pCount for number of BW counters? (i.e, currently we have 4 HBM counters and all are 32 bit so any of them could overflow). so we could have pCount for num counters and then user provides an array of pCount size so we can set num valid bits per counter

Reply:
do we also need new API with the duration input arg which could hide the complexity of rollover detection?

Reply:
We are not exposing counter per HBM stack, the pcode will detect the rollover for individual counter and add together to return single read/write counter for HBM

Reply:
Duration count is a temporary fix in the experiment API until the pcode 64 bit counter fix is available

Reply:
@aravindksg is the above satisfactory?

Reply:
@mraghuwa does above seem fine given pcode should detect individual counter overflows?

Reply:
It does for PVC program

Reply:
@saik-intel please check above and confirm if addition to expose counter width is fine?